source/Mission.h

```diff
@@ -204,7 +208,16 @@ class Mission {
 	bool hasFailed = false;
 	bool isVisible = true;
 	bool hasPriority = false;
+
 	bool isMinor = false;
+	// `order` is a sort override, missions are sorted by order first then alphabetically within each order level.
+	// `minor` missions are implemented by discarding from the sorted beginning, thus negative means decreased priority.
+	int order = 0;
+	// TODO: Storage of missions in UniverseObjects sorts during load, which is now redundant.
+	//       But: On-demand sort may be faster if the missions come 99% pre-sorted?
+	// TODO: Test that the order field can actually reorder jobs over strictly alphabetic.
+	// TODO: Update the wiki.
```

Comment:
Realize that the size of the availableMissions list after CreateMissions runs is likely only has a single digit length, and that CreateMissions is called during a non-performance intensive point (when you're landing on a planet and therefore when all ship AI, projectile movements, and collision detection are done), so the difference between pre-sorting the missions when the game loads and just sorting the availableMissions when the list is populated is absolutely miniscule.